lambda_strip_docx/test_clean_pdf.py

```diff
@@ -0,0 +1,35 @@
+import clean_pdf
+
+def pad_name(name):
+    split_name_ints = list(name)
+    split_name_chars = [chr(x) for x in split_name_ints]
+    unicode_name = "\x00" + "\x00".join(split_name_chars)
+    return unicode_name.encode('utf-8')
+
+def check_sample_pdf_metadata_is_stripped(pdf_bytes):
+    names = [b"Alice Johnson", b"Bob Smith", b"Carol Davis"]
+    padded_names = [pad_name(name) for name in names]
+    for padded_name in padded_names:
+        assert padded_name not in pdf_bytes
+
+def test_cleaning():
```

Comment:
whats is the point of testing `check_sample_pdf_metadata_is_stripped` if not used anywhere else?